clang/docs/ReleaseNotes.rst

```diff
@@ -302,6 +302,11 @@ Attribute Changes in Clang
   to reduce the size of the destroy functions for coroutines which are known to
   be destroyed after having reached the final suspend point.
 
+- On targets with C++ ABI other than Microsoft, Clang now supports
```

Comment:
This should just say 'Itanium' rather than talking around it (other than Microsoft).